cmd/juju/ssh/ssh_machine.go

```diff
@@ -90,7 +91,7 @@ func (t *resolvedTarget) isAgent() bool {
 }
 
 // sshPort is the TCP port used for SSH connections.
-const sshPort = 22
+var sshPort = 22
```

Comment:
This is a smell. 

You should pass the new value through the functions, not weaken the modeling.

Reply:
Hey simon! Thanks for the feedback, I have updated accordingly :)